<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Admin_SRV-PC" ignored="0" name="Testch" tests="15" failures="15" timestamp="2022-05-18T21:35:44 IST" time="1.144" errors="0">
  <testcase name="@AfterTest teardown" time="0.0" classname="tests.Datautilslogindemo">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at tests.Datautilslogindemo.teardown(Datautilslogindemo.java:112)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- @AfterTest teardown -->
  <testcase name="finaldata" time="0.016" classname="tests.Datautilslogindemo">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at tests.Datautilslogindemo.finaldata(Datautilslogindemo.java:56)
at java.util.ArrayList.forEach(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- finaldata -->
  <testcase name="finaldata" time="0.0" classname="tests.Datautilslogindemo">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at tests.Datautilslogindemo.finaldata(Datautilslogindemo.java:56)
at java.util.ArrayList.forEach(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- finaldata -->
  <testcase name="finaldata" time="0.0" classname="tests.Datautilslogindemo">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at tests.Datautilslogindemo.finaldata(Datautilslogindemo.java:56)
at java.util.ArrayList.forEach(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- finaldata -->
  <testcase name="finaldata" time="0.0" classname="tests.Datautilslogindemo">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at tests.Datautilslogindemo.finaldata(Datautilslogindemo.java:56)
at java.util.ArrayList.forEach(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- finaldata -->
  <testcase name="finaldata" time="0.0" classname="tests.Datautilslogindemo">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at tests.Datautilslogindemo.finaldata(Datautilslogindemo.java:56)
at java.util.ArrayList.forEach(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- finaldata -->
  <testcase name="finaldata" time="0.0" classname="tests.Datautilslogindemo">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at tests.Datautilslogindemo.finaldata(Datautilslogindemo.java:56)
at java.util.ArrayList.forEach(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- finaldata -->
  <testcase name="finaldata" time="0.0" classname="tests.Datautilslogindemo">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at tests.Datautilslogindemo.finaldata(Datautilslogindemo.java:56)
at java.util.ArrayList.forEach(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- finaldata -->
  <testcase name="finaldata" time="0.0" classname="tests.Datautilslogindemo">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at tests.Datautilslogindemo.finaldata(Datautilslogindemo.java:56)
at java.util.ArrayList.forEach(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- finaldata -->
  <testcase name="finaldata" time="0.0" classname="tests.Datautilslogindemo">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at tests.Datautilslogindemo.finaldata(Datautilslogindemo.java:56)
at java.util.ArrayList.forEach(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- finaldata -->
  <testcase name="finaldata" time="0.0" classname="tests.Datautilslogindemo">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at tests.Datautilslogindemo.finaldata(Datautilslogindemo.java:56)
at java.util.ArrayList.forEach(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- finaldata -->
  <testcase name="finaldata" time="0.0" classname="tests.Datautilslogindemo">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at tests.Datautilslogindemo.finaldata(Datautilslogindemo.java:56)
at java.util.ArrayList.forEach(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- finaldata -->
  <testcase name="finaldata" time="0.0" classname="tests.Datautilslogindemo">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at tests.Datautilslogindemo.finaldata(Datautilslogindemo.java:56)
at java.util.ArrayList.forEach(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- finaldata -->
  <testcase name="finaldata" time="0.0" classname="tests.Datautilslogindemo">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at tests.Datautilslogindemo.finaldata(Datautilslogindemo.java:56)
at java.util.ArrayList.forEach(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- finaldata -->
  <testcase name="finaldata" time="0.0" classname="tests.Datautilslogindemo">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at tests.Datautilslogindemo.finaldata(Datautilslogindemo.java:56)
at java.util.ArrayList.forEach(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- finaldata -->
  <testcase name="finaldata" time="0.016" classname="tests.Datautilslogindemo">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at tests.Datautilslogindemo.finaldata(Datautilslogindemo.java:56)
at java.util.ArrayList.forEach(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- finaldata -->
</testsuite> <!-- Testch -->
